sdk/identity/azure-identity/azure/identity/_credentials/default.py

```diff
@@ -76,7 +80,7 @@ class DefaultAzureCredential(ChainedTokenCredential):
         Directory work or school accounts.
```

Comment:
Add `:keyword  str workload_identity_client_id` somewhere in here And similarly in the aio version?